server/publications/collections/accounts.js

```diff
@@ -45,6 +45,22 @@ Meteor.publish("Accounts", function (userId) {
 });
 
 /**
+ * Single account
+ * @params {String} userId -  id of user to find
+ */
+Meteor.publish("Accounts.single", function (userId) {
```

Comment:
While I love the `Product/Products` naming convention in theory, it does create a lot of opportunities for bugs. I do sort of like the dot naming convention because it doesn't result in long, sometimes hard to read camel-case names. It's also the style shown in the Meteor guide.